src/plugins/formulas/formulas.js

```diff
@@ -379,6 +390,7 @@ export class Formulas extends BasePlugin {
 
       this.#shouldSuspendRenders = true;
       this.engine.setCellContents(address, valueHolder.value);
+      this.validateCell(address);
```

Comment:
Is this necessary? The exact same line will be called in `onEngineValuesUpdated`, as `setCellContents` triggers it already.

Reply:
Old code, before refactor, I will remove.